recipes/scrape-search-engine/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "scrape-search-engine" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c1d6093053e54558d1a883608285e143363512df123699647a5353ee6e068c27
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - bs4
+    - requests
+    - python >=3.6
+
+test:
+  imports:
+    - ScrapeSearchEngine
```

I noticed that this also installs a `SearchEngine` module. You should check that it imports too, in addition to `ScrapeSearchEngine`.
```suggestion
    - ScrapeSearchEngine
    - SearchEngine
```

---

Sorry, I missed that this was importing the package and not the module. Not that it's really relevant to conda-forge and the packaging itself, but I'm surprised that importing the package works because it doesn't have an `__init__.py`, and I thought Python required that to recognize it as a package. Ah, I guess this makes it an ["implicit namespace package"](https://packaging.python.org/en/latest/guides/packaging-namespace-packages/). It doesn't seem like that's what you'd want, but I'm not here to review the code, just the packaging.

Given that, then, I'd suggest a `command` test of `python -c "from ScrapeSearchEngine.ScrapeSearchEngine import Google"` to ensure that at least some code gets imported. That will check that you have the dependencies correct at least.